### PR TITLE
Feature/79 metadata upgrade

### LINKED ISSUE #79: Upgrade to new protocols packages

### PR DESCRIPTION
**Pull request by piersss**
# Description

Fixes #79 

Removed CollectionToken and replaced with CollectionTokenInfo from @airswap protocols.

# Checklist:

- [x] I have read and understood [CONTRIBUTING.MD](https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md)
- [x] I have included the ticket number in my PR
- [x] I have performed a self-review of my own code
- [x] I have commented the parts where my code is hard to understand

### CODE REVIEW

File: src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const ConnectedNftDetailWidget: FC<ConnectedNftDetailWidgetProps> = ({ library,
           />
           <NftDetailSaleInfo
             price={BigNumber.from('0')}
-            symbol={tokenMeta.symbol}
+            symbol={tokenMeta.name}
```
Comment (Goddak) [inline src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx:63]: What is the reason for this change?

**Review by Goddak — COMMENTED** [review]
Seems to be working, not sure why we changed symbol for name though?

File: src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx (hunk 1 above)
Comment (piersss) [reply src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx:63]: The CollectionTokenInfo from airswap package will not have `symbol` anymore, just `name`.

**Comment by piersss** [comment]
> Seems to be working, not sure why we changed symbol for name though?

Symbol came from `TokenInfo`. Since `CollectionTokenInfo` is purely for nft's we can leave out `symbol`, since nft's don't use symbols; they have names and id.

File: src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx (hunk 1 above)
Comment (Goddak) [reply src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx:63]: Where do we plan to get the symbol from moving forwards?

Comment (piersss) [reply src/widgets/NftDetailWidget/subcomponents/ConnectedNftDetailWidget/ConnectedNftDetailWidget.tsx:63]: Only erc-20 (`TokenInfo`) token will have a symbol

**Review by Goddak — APPROVED** [review]
:+1:
